cmp/grid/columns/Column.js

```diff
@@ -194,7 +194,6 @@ export class Column {
                 resizable: this.resizable,
                 sortable: this.sortable,
                 suppressMovable: !this.movable,
-                lockPinned: true, // Block user-driven pinning/unpinning - https://github.com/exhi/hoist-react/issues/687
```

Comment:
We should probably still lock via this setting when mobile - in that case, we don't want the user-initiated drag-to-edge gesture to interfere with the more managed (and I think appropriate) pin-first-col setting available via the chooser.

(I also tried dragging cols to pin on Chrome w/mobile emulation and it froze up - might just be an emulator thing but was enough weirdness to convince me we don't need this feature on mobile at the moment.)